app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala

```diff
@@ -216,8 +217,15 @@ class MovementsRepositoryImpl @Inject() (
         obs
           .headOption()
           .map {
-            case Some(opt) => Right(opt)
-            case None      => Left(DocumentNotFound(s"No movement found with the given id: ${movementId.value}"))
+            case Some(opt) =>
```

Comment:
I don't think so it is required? As this is for get call from common-transit-convention-traders?